docs/guide/testrunner/organizesuite.md

```diff
@@ -137,6 +137,27 @@ $ grep -r -l --include "*.js" "myText" | wdio wdio.conf.js
 
 _**Note:** This will_ not _override the `--spec` flag for running a single spec._
 
+## Exclude Selected Tests
+
+When needed, if you need to exclude particular spec file(s) from a run, you can use the `--exclude` parameter (Mocha, Jasmine). For example if you want to exclude your login
```

Comment:
Why only Mocha and Jasmine? In Cucumber with `--spec` you can select feature files, why not be able to exclude them?

Reply:
typo, will fix...